server/container_create_linux.go

```diff
@@ -758,6 +758,12 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrfactory.Cont
 		ctr.DisableFips(),
 	)
 
+	if ctr.DisableFips() && sb.Annotations()[crioann.DisableFIPSAnnotation] == "true" {
+		if err := disableFipsForContainer(ctr, containerInfo.RunDir); err != nil {
+			return nil, fmt.Errorf("failed to disable FIPS for container %s: %w", containerID, err)
```

Comment:
We say "pod" everywhere else, here we say "container". Where does the bind mount over go? Container, right?

Reply:
In this context, since this logic is part of the `createSandboxContainer` function, it makes sense to use the term 'container.' however, it's important to understand that containers are ultimately part of a pod.